src/corner-kick/src/components/modules/Control.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+
+import { Subscribe } from 'unstated';
+
+import { TopicPublisher } from '~/components/containers/TopicPublisher';
+import { ITurtleCmdVel } from '~/types/standardTopics';
+
+// ROS Messages
+const UP = {
+    angular: {
+        x: 0,
+        y: 0,
+        z: 0,
+    },
+    linear: {
+        x: 2,
+        y: 0,
+        z: 0,
+    },
+};
+
+const DOWN = {
+    angular: {
+        x: 0,
+        y: 0,
+        z: 0,
+    },
+    linear: {
+        x: -2,
+        y: 0,
+        z: 0,
+    },
+};
+
+const LEFT = {
+    angular: {
+        x: 0,
+        y: 0,
+        z: 2,
+    },
+    linear: {
+        x: 0,
+        y: 0,
+        z: 0,
+    },
+};
+
+const RIGHT = {
+    angular: {
+        x: 0,
+        y: 0,
+        z: -2,
+    },
+    linear: {
+        x: 0,
+        y: 0,
+        z: 0,
+    },
+};
```

Comment:
I think we can probably remove these if we're pushing past the MVP stage. Just as a side note though, why aren't these typed?